app/views/devise/sessions/new.html.haml

```diff
@@ -1,16 +1,41 @@
-%article.content-listing
-  %h2 Log in
-  %section.form-wrapper
+%article.sign-in
+  %h1 Sign in &mdash; GOV.AU Author
```

Comment:
Did this new title come from somewhere? I think it can just be "Sign in" without "GOV.AU Author". Author is just one role. Similar how the signup page just says "Sign up".